### PR TITLE
Basic bot interaction

### PR DESCRIPTION
**Pull request by Stingzzy**

### CODE REVIEW

**Review by tinoaxmskp — APPROVED** [review]

File: bot/bot_interaction.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from typing import Final
+
+import telebot
```
Comment (rubinnum) [inline bot/bot_interaction.py:3]: this package has to be added to requirements.txt file

File: bot/bot_interaction.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+from telebot import types
```
Comment (rubinnum) [inline bot/bot_interaction.py:4]: since you already imported _telebot_ package, remove this unnecessary import (later explicitly use _types_ module from _telebot_ package)

File: bot/constants.py [hunk 3]
```diff
@@ -8,3 +8,6 @@
 GPT_MODEL = 'gpt-3.5-turbo'
 ORGANIZATION_ID = 'org-XwMNyaooe9J64WbgekZen3L0'
 PROJECT_ID = 'proj_GZHV06BPXtu8fHt2xwoEuhmp'
+
+# Telegram Token
+BOT_TOKEN = '7144215394:AAGJXRA-PYsuczsq-AiPmLuxVBKkdM27lIw'
```
Comment (rubinnum) [inline bot/constants.py:13]: BOT_TOKEN should not be displayed and pushed to the remote repository due to security reasons. Assign the value of this variable to 'BOT_TOKEN' (as I've done it with OPENAI_API_KEY) and save the token itself somewhere locally for yourself, to past it later whenever you need to use it (but without pushing these changes to remote repository)

File: bot/bot_interaction.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+from constants import BOT_TOKEN
+
+bot = telebot.TeleBot(BOT_TOKEN, parse_mode=None)
+BOT_USERNAME: Final = '@game_whisper_bot'
```
Comment (rubinnum) [inline bot/bot_interaction.py:9]: Remove this variable since it's not used anywhere

File: bot/bot_interaction.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+from typing import Final
+
+import telebot
+from telebot import types
+
+from constants import BOT_TOKEN
+
+bot = telebot.TeleBot(BOT_TOKEN, parse_mode=None)
```
Comment (rubinnum) [inline bot/bot_interaction.py:8]: Remove the parse_mode parameter here since it's gonna be None by default

File: bot/bot_interaction.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+BOT_USERNAME: Final = '@game_whisper_bot'
+
+
+@bot.message_handler(commands=['start', 'help'])
+def send_welcome(message):
+    bot.reply_to(message, ' Welcome to Gaming Whisper')
+    markup = types.ReplyKeyboardMarkup(row_width=2)
+    button_1 = types.InlineKeyboardButton('Game Profile')
```
Comment (rubinnum) [inline bot/bot_interaction.py:16]: Give an appropriate name for this button here (something like game_profile_button), because if it's gonna be required later in the code, you would type _button_1_ and collaborators would have no idea what kind of button this is exactly. Same applies to the button_2. That's a bad naming

File: bot/bot_interaction.py [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+from constants import BOT_TOKEN
+
+bot = telebot.TeleBot(BOT_TOKEN, parse_mode=None)
+BOT_USERNAME: Final = '@game_whisper_bot'
+
+
+@bot.message_handler(commands=['start', 'help'])
+def send_welcome(message):
+    bot.reply_to(message, ' Welcome to Gaming Whisper')
```
Comment (rubinnum) [inline bot/bot_interaction.py:14]: Remove unnecessary whitespace from second parameter

File: bot/bot_interaction.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    markup = types.ReplyKeyboardMarkup(row_width=2)
+    button_1 = types.InlineKeyboardButton('Game Profile')
+    button_2 = types.InlineKeyboardButton('Personal Recommendation')
+    markup.add(button_1, button_2)
+    bot.send_message(message.chat.id, "Your choice for today:", reply_markup=markup)
+
+
+@bot.message_handler(func=lambda m: True)
```
Comment (rubinnum) [inline bot/bot_interaction.py:22]: Change the name of a function to message so it would be more clear

File: bot/bot_interaction.py [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+def echo_all(message):
+    bot.reply_to(message, message.text)
+
+
+bot.infinity_polling()
```
Comment (rubinnum) [inline bot/bot_interaction.py:27]: Wrap the polling into `if __name__ == '__main__':`

**Review by rubinnum — CHANGES_REQUESTED** [review]
Commends added
